index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+
+    <title>Awesome Books</title>
+
+</head>
+<body class="body">
+    <nav>
+        <h1>Awesome Books</h1>
+        <p id ="demo"></p>
+    </nav>
+ <section class="books">
+    <div class="container">
+        <table class="table table-striped mt-5">
+            <thead>
+                <tr>
+                    <th>Title</th>
+                    <th>Author</th>
+                    <th>Action</th>
+                </tr>
+            </thead>
+            <tbody id="book-list"></tbody>
+        </table>
+    </div>
+</section>
+    <form id="form">
+        <p></p>
+        <input id="book-title" type="text" placeholder="Title" required  class="input"/>
+        <p></p>
+        <input id="author" type="name" placeholder="Author" required class="input"/>
+        <br>
+        <button type="submit" id="add">Add</button>
+      </form>
+<!-- <script src="app.js"></script> -->
```

Comment:
- [ ] As suggested by the previous reviewer, please remove all the commented-out code. This will make your codebase looks better and cleaner